app/pages/500.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import StaticLayout from "components/Layout/StaticLayout";
+import getConfig from "next/config";
+import Link from "next/link";
+import { Button } from "@button-inc/bcgov-theme";
+import { useRouter } from "next/router";
+import Alert from "@mui/material/Alert";
```

Comment:
I just didn't realize it existed because I was looking [here](https://github.com/button-inc/service-development-toolkit/blob/develop/docs/components.md). I'll check storybook in the future